languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -1536,7 +1536,7 @@ though, and has since been slightly modified)-->
           <marker><token regexp="yes">choeurs?|chœurs?</token></marker>
         </pattern>
         <message>Vouliez-vous écrire <suggestion>cœur</suggestion> ?</message>
-        <example correction="cœur">Il sait tout par <marker>chœur</marker>.</example>
+        <example correction="cœur">Il le sait tout par <marker>chœur</marker>.</example>
```

Comment:
I don't think that there was anything wrong in the original example.

Reply:
Thanks for checking — this was literally just a `[fr] test pull request` :)